relays/utils/src/lib.rs

```diff
@@ -51,7 +52,7 @@ pub mod relay_loop;
 pub trait BlockNumberBase:
 	'static
 	+ From<u32>
-	+ Into<u64>
+	+ UniqueSaturatedInto<u64>
```

Comment:
This is now used instead of simple `.into()` in two places: (1) for metrics (2) when we talk about transaction mortality. But we talk about block number here and it is limited by 2^64, which is imo enough. In worst case we'll have: (1) an alert when there are more than 2^64 blocks and (2) transaction which dies after 2^64 blocks (but if you're crafting such tx, you probably mean immportal tx)